### DISCUSSION TITLE
Why do PR's automatically go to deepskies/proj-clusters-sz-sbi?

### DISCUSSION THREAD

**Discussion by bnord** [deepskies/DeepSZSim#56] [Q&A] [answered]
@voetberg @cavestruz 

I forked the current repository `SimSZ` from https://github.com/deepskies/proj-clusters-sz-sbi. 

When I try to do a PR from any branch, it automatically wants to go to `main` of `proj-clusters-sz-sbi`.

But, we just want this current repo to be the one we use and publish. We only want to PRs to go to `SimSZ`, not `proj-clusters-sz-sbi`.

Do you know how to change this default behavior when PRs are started? Is it possible?

**Comment by voetberg** [comment]
@bnord 

Is this happening just when you make prs from a gui? or when you're doing it from the web client? If it's the first, you'll just need to change your remote with `git remote add upstream https://github.com/deepskies/SimSZ/`

But if you want to completely remove it from the original repo, it's probably best to clone `proj-clusters-sz-sbi` (Or `SimSZ` even) and push it up to a new place without using forking.

**Reply by bnord** [reply]
I'm using the github client for most my work right now. 

When you say GUI, you mean something like VScode on my local machine?

**Reply by voetberg** [reply, marked as answer]
Yep, that's what I mean. 

If that's the case though, it's probably best to just clone this repo and then push it up as a different remote entirely, removed from the original repo.

**Reply by bnord** [reply]
ah okay. 

will do 

sad face
